README.md

```diff
@@ -14,7 +14,7 @@ For more details and some background information about this project, see the blo
 
 ## Prerequisites
 
-* Run `npm install` to install dependencies required by the node script
+* Run `sudo apt-get install -y jq` to install [jq](https://stedolan.github.io/jq/) required by the bash script
```

Comment:
Could you add a note that if on MacOS, jq can be installed with `brew install jq`?

Also, a new dependency is that openssl is installed and found from the path. This could be mentioned.

Reply:
Done, I've added link to jq's download page so that appropriate instructions can be followed for each OS.